en/guide/modules.md

```diff
@@ -3,69 +3,56 @@ title: Modules
 ---
 
-> Modules are Nuxt.js extensions which can extend it's core functionality and add endless integrations. 
+> Modules are Nuxt.js extensions which can extend it's core functionality and add endless integrations.
 
 ## Introduction
 
-While developing production grade application using Nuxt, you will find out soon that nuxt core functionalities are not enough
-and writing configs and plugins for each project is a repetitive, boring and time consuming job.
-Also adding every new feature into Nuxt is not possible as it would make it a fat framework.
-   
-This was the reason Nuxt introduces a higher order modular system to easily extend the core.
-Modules are basically **functions** which are called sequentially when booting Nuxt and core awaits for all of them
-to be finished before continue it's job. So they have the chance to customize almost any aspect of Nuxt and thanks to modular design of nuxt itself and Webpack [Tapable](https://github.com/webpack/tapable) technology they can also register hooks
-for certain entry points like builder initialization.
+While developing production grade application using Nuxt, you will find out soon that Nuxt core functionalities are not enough and writing configs and plugins for each project is a repetitive, boring and time consuming job. Also adding every new feature into Nuxt is not possible as it would make it a fat framework.
 
-Another point of using modules is that they can be refactored and packaged out of the project and released as NPM packages
-so you can share and use high quality integration and solutions from nuxt community with no pain! You might interested in modules if you:
+This was the reason Nuxt introduces a higher order modular system to easily extend the core. Modules are basically **functions** which are called sequentially when booting Nuxt and core awaits for all of them to be finished before continue it's job. So they have the chance to customize almost any aspect of Nuxt and thanks to modular design of Nuxt itself and webpack [Tapable](https://github.com/webpack/tapable) technology they can also register hooks for certain entry points like builder initialization.
 
-- Are a member of an **agile team** that want to set up your project instantly and avoid **re-inventing** the wheel for common tasks like google-analytics for your new project.
+Another point of using modules is that they can be refactored and packaged out of the project and released as npm packages so you can share and use high quality integration and solutions from Nuxt community with no pain! You might interested in modules if you:
+
+- Are a member of an **agile team** that want to set up your project instantly and avoid **re-inventing** the wheel for common tasks like Google Analytics tools for your new project.
 - Are an **enterprise** company which **quality** and **reusability** is important for your projects.
```

Comment:
I'm not sure @Atinux but the usage of enterprise company is not redundant? Maybe documentation would say **enterprise** or **company**?
I always thinking this word was switchable but not additives?

I'm wrong XD?